data/random-teams.ts

```diff
@@ -341,7 +341,8 @@ export class RandomTeams {
 				counter.add('damage');
 				counter.damagingMoves.add(move);
 			} else {
-				// Are Physical/Special/Status moves:
+				// Are Physical/Special/
+				moves:
```

Comment:
```suggestion
				// Are Physical/Special moves:
```
I assume the new line was an accident?